### PR TITLE
Update/segun/single order details

### PR DESCRIPTION
**Pull request by Segun-Ade**
#### What does this PR achieve?
Implements few minor changes

### CODE REVIEW

File: src/components/userDashboard/components/orders/components/orderDetails.jsx [hunk 1]
```diff
@@ -34,16 +44,25 @@ function ClothItem(item) {
...
     return (
-        <Descriptions title="Vendor Info" layout="vertical" key={orderById.orderID} bordered>
+        <Descriptions
+            className="order-details-view"
+            title="Vendor Info"
+            layout="vertical"
+            key={singleOrder.orderID}
+            bordered
+        >
             <Descriptions.Item label="Phone">
```
Comment (justiceotuya) [inline src/components/userDashboard/components/orders/components/orderDetails.jsx]: try to destructure 
`const {Item} = Descriptions` 
`const {orderItems, orderID, address, createdAt} = singleOrder` 
`const {phoneNumber} = userInfo`

Comment (Segun-Ade) [reply src/components/userDashboard/components/orders/components/orderDetails.jsx]: Ok

File: src/components/userDashboard/components/make complaints/components/makeComplaint.jsx [hunk 2]
```diff
@@ -93,7 +93,7 @@ const MakeComplaint = ({ form }) => {
                         <Form.Item label="Orderid" hasFeedback>
                             {getFieldDecorator('orderid', {
                                 rules: [
-                                    { message: { ORDER_ID_MESSAGE }, required: true },
+                                    { message: 'Order Id can not be blank', required: true },
```
Comment (edbella) [inline src/components/userDashboard/components/make complaints/components/makeComplaint.jsx]: Why did you change back to strings?

File: src/components/userDashboard/components/orders/actionTypes.js [hunk 3]
```diff
@@ -3,5 +3,7 @@ const actionTypes = {
     GET_SINGLE_ORDER_DETAILS: 'GET_SINGLE_ORDER_DETAILS',
     ORDER_REQUEST_FAILURE: 'ORDER_REQUEST_FAILURE',
     ORDER_REQUEST_SUCCESS: 'ORDER_REQUEST_SUCCESS',
+    REQ_SINGLE_ORDER_FAILED: 'REQ_SINGLE_ORDER_FAILED',
```
Comment (edbella) [inline src/components/userDashboard/components/orders/actionTypes.js]: Not sure this naming really cuts it. Try something else that makes it shorter and if not you can leave it be

File: src/components/userDashboard/components/make complaints/components/makeComplaint.jsx (hunk 2 above)
Comment (Segun-Ade) [reply src/components/userDashboard/components/make complaints/components/makeComplaint.jsx]: It was crashing the app. The rules array doesn't allow such expressions.
